content/academy/become_an_apify_expert/solutions/integrating_webhooks.md

```diff
@@ -171,11 +171,33 @@ Click **Save**, then run the Amazon **demo-actor** again.
 
 If everything worked, then at the end of the **demo-actor**'s run, we should see this within the **Integrations** tab:
 
-![Webhook succeeded]({{@asset pro_apify_scraper_development/solutions/images/webhook-succeeded.webp}})
+![Webhook succeeded]({{@asset become_an_apify_expert/solutions/images/webhook-succeeded.webp}})
 
 Additionally, we should be able to see that our **filter-actor** was run, and have access to its dataset:
 
-![Dataset preview]({{@asset pro_apify_scraper_development/solutions/images/dataset-preview.webp}})
+![Dataset preview]({{@asset become_an_apify_expert/solutions/images/dataset-preview.webp}})
+
+## [](#quiz-answeers) Quiz answers 📝
+
+- How do you allocate more CPU for an actor's run?
```

Comment:
I'm not a fan of how this looks when rendered. Would you consider just making the questions bold?

<img width="721" alt="Screenshot 2022-04-22 at 15 41 56" src="https://user-images.githubusercontent.com/38246758/164726214-28918d72-4c28-4b5c-a51c-080892b066c6.png">

What do you think?

Reply:
I love this. Much better

Reply:
I am going to make the _A:_ italic. Please don't kill me